csgo.yaml

```diff
@@ -82,7 +82,7 @@ Resources:
           echo "bot_quota 0" >> /Steam/csgo-ds/csgo/cfg/gamemode_armsrace.cfg
           echo "mp_solid_teammates 1" >> /Steam/csgo-ds/csgo/cfg/server.cfg
           echo "mp_solid_teammates 1" >> /Steam/csgo-ds/csgo/cfg/gamemode_armsrace.cfg
-          /Steam/csgo-ds/srcds_run -game csgo -console -usercon +game_type 1 +game_mode 0 +mapgroup mg_armsrace +map de_safehouse -nobots +sv_setsteamaccount ${GSLT}
+          /Steam/csgo-ds/srcds_run -game csgo -console -usercon +game_type 1 +game_mode 0 +host_workshop_collection 843890618 +workshop_start_map 175480758 -nobots +sv_setsteamaccount ${GSLT}
```

Comment:
`+game_type 1 +game_mode 0`  will still fun the server in "ArmsRace" mode. is that what you want?